commander/src/commands/generate/plugin.ts

```diff
@@ -41,7 +41,8 @@ export default class PluginCommand extends BaseBootstrapCommand {
 		// validate folder name to not include camelcase or whitespace
 		const regexWhitespace = /\s/g;
 		const regexCamelCase = /^([a-z]+)(([A-Z]([a-z]+))+)$/;
-		if (regexCamelCase.test(alias) || regexWhitespace.test(alias)) {
+		const regexAlphabets = /[^A-Za-z]/;
+		if (regexCamelCase.test(alias) || regexWhitespace.test(alias) || regexAlphabets.test(alias)) {
```

Comment:
@shuse2 I tested it with several values and it didn't work well and I think these valid values. 

```
> alias = "my-plugin"
'my-plugin'
> regexCamelCase.test(alias) || regexWhitespace.test(alias) || regexAlphabets.test(alias)
true
> alias = "my_plugin"
'my_plugin'
> regexCamelCase.test(alias) || regexWhitespace.test(alias) || regexAlphabets.test(alias)
true
```


Can we also expand the scope of the issue a bit further to also  to? 

- validate plugin class names and 
- module names and class names and
- application label

Reply:
right now, what we allow in the class name seems to be with `regexCamelCase.test(alias) || regexWhitespace.test(alias) || regexAlphabets.test(alias)` rule, and i think it's better to be consistent with the module naming?